src/main/java/duke/command/BetweenCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+
+import duke.exception.DukeIllegalArgumentException;
+import duke.task.TaskList;
+import duke.util.Printable;
+
+/**
+ * Represents a handler for the between command.
+ */
+public class BetweenCommand extends Command {
+    /**
+     * Creates a handler for the between command.
+     * @param args Arguments supplied to the command handler.
+     */
+    BetweenCommand(String args) {
+        super(args);
+    }
+
+    @Override
+    public boolean execute(Printable linePrinter, TaskList taskList) throws DukeIllegalArgumentException {
+        final String[] argParts = args.split(" and ");
+        if (argParts.length < 2) {
+            throw new DukeIllegalArgumentException("Not in the format <date> and <date>");
+        }
+        final LocalDateTime start = parseDateTime(argParts[0]);
+        final LocalDateTime end = parseDateTime(argParts[1]);
```

Comment:
Should 'start' and 'end' be renamed as 'START' and 'END' instead?

Reply:
I believe that this is a local final variable rather than a constant (defined by the presence of `static final`), so it should not follow the screaming snake case.